WooCommerce/WooCommerceTests/ViewRelated/Orders/Issue Refund/IssueRefundViewModelTests.swift

```diff
@@ -158,4 +179,60 @@ final class IssueRefundViewModelTests: XCTestCase {
         // 23.00(current products) + 7.52 (shipping) = 30.62
         XCTAssertEqual(viewModel.title, "$30.62")
     }
+
+    func test_viewModel_starts_with_zero_count_label() {
+        // Given
+        let currencySettings = CurrencySettings()
+        let items = [
+            MockOrderItem.sampleItem(itemID: 1, quantity: 3, price: 11.50),
+            MockOrderItem.sampleItem(itemID: 2, quantity: 2, price: 12.50),
+            MockOrderItem.sampleItem(itemID: 3, quantity: 1, price: 13.50),
+        ]
+        let order = MockOrders().makeOrder(items: items)
+
+        // When
+        let viewModel = IssueRefundViewModel(order: order, currencySettings: currencySettings)
+
+        // Then
+        let selectedItemsTitle = String(format: NSLocalizedString("%d items selected", comment: ""), 0)
```

Comment:
nit: I think it's reasonable to make `IssueRefundViewModel.Localization` internal instead of private so that we can refer to the Localization constants in unit tests in case they change in the future? (Same comment for other `NSLocalizedString`s in this test class)

Reply:
Hmm, I normally don't like to expose a type's implementation details just for testing but I also don't like to write twice a localized string 🤔 

Maybe we should make a small vote within the team and decide what's preferred?